components/crypto/crypto-service-impl/src/main/kotlin/net/corda/crypto/service/impl/bus/CryptoRekeyBusProcessor.kt

```
@@ -81,8 +82,10 @@ class CryptoRekeyBusProcessor(
         logger.debug("processing $request")
         require(request != null)
 
-        if (!hasPreviousRotationFinished()) {
-            logger.info("A key rotation is already ongoing, ignoring request to start new one.")
+        try {
```

This is picking up a programming error, so it's unusual to suppress it like this. Actually now I look at the code, why does any of it work like this?

The problem seems to be in `CryptoProcessorImpl`, because state manager requires the boot config, but we should not have let that propagate all the way down here, just to throw/ignore it. It's not part of this ticket, but it might be better if you want to handle this, to handle it up there and remove the nullability of state manager everywhere else.